python/paddle/trainer/config_parser.py

```diff
@@ -1006,6 +1006,17 @@ def TestData(data_config, async_load_data=None):
                        " Data definition")
         g_config.test_data_config.async_load_data = async_load_data
 
+'''
+caffe_mode: compute the output size using floor instead of ceil,
+            which is consistent of caffe and CuDNN's convention.
+'''
+def cnn_output_size(img_size, filter_size, padding, stride, caffe_mode):
```

Comment:
`caffe_mode` can be set true by default, because it only set false once in exconv layer. 

Looks good to me.  @hedaoyuan  please help to review again.